content/docs/engine-dev/tools/project-manager/bootup-sequence/_index.md

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
Unless you plan on having multiple topics about the bootup sequence, this file does not need to be in a subdirectory, and you can name it `bootup-sequence.md`.